sql/2019/12_Mobile_Web/12_14.sql

```diff
@@ -0,0 +1,34 @@
+#standardSQL
+
+# input attributes
+
+# dataset: `httparchive.pages.2019_07_01_mobile`
+# sample: `httparchive.almanac.pages_desktop_1k`
+
+CREATE TEMPORARY FUNCTION parseStructuredData(payload STRING)
+RETURNS ARRAY<STRING> LANGUAGE js AS '''
+  var attrs = [];
+  try {
+    var $ = JSON.parse(payload);
+    var almanac = JSON.parse($._almanac);
+    almanac['input-elements'] && almanac['input-elements'].map(function(node) {
```

Comment:
Nit: These two functions use `map` but don't transform the items. Use `forEach` instead.